tests/integration/servicecatalog/service_test.go

```diff
@@ -84,11 +84,11 @@ var _ = Describe("odoServiceE2e", func() {
 			// Ping said URL
 			helper.HttpWaitFor(routeURL, "Spring Boot", 90, 1)
 
-			// Delete the component
```

Comment:
@mik-dass What is the need of removing ```helper.CmdShouldPass("odo", "delete", "sb-app", "-f")```

Reply:
Because in 
```
JustAfterEach(func() {
			helper.DeleteProject(project)
```
we are deleting the project which internally calls list to display the info of the components. According to this PR, `component list` will show a component without the need for pushing. Thus the project delete call will error out. Thus I moved it after the service delete call as even the service call needs the config to figure out the project.

Reply:
I'm confused.  Are you saying that `odo project delete` will fail if there are no components in the project? That is not right.

Reply:
TBH I don't remember why I wrote the project delete thing, I was facing some error there though :D But the main reason is that service delete will list the components and since the list call will pick up the component from the config (It was getting deleted before the service delete) and try to find the deployment config accordingly and thus will fail.